src/tsolvers/lasolver/LASolver.cc

```diff
@@ -244,7 +243,7 @@ LVRef LASolver::getVarForLeq(PTRef ref) const {
 }
 
 LVRef LASolver::getLAVar_single(PTRef expr_in) {
-
+    if (logic.isTimesNonlin(expr_in) || logic.isRealDiv(expr_in)) throw NonLinException(logic.pp(expr_in));
```

Comment:
Why do check for real div here? I thought we agreed that div should never occur at this point.

Reply:
The problem here is that `mkRealDiv` may be called for the function declaration, and then this function may be used from outside while assigning the variables in the divisor with  constants, so it could be still handled with LASolver...

Reply:
Ok, Real division is no longer supported